operator/informer_customcache.go

```diff
@@ -193,7 +193,7 @@ func (c *CustomCacheInformer) errorHandler(ctx context.Context, err error) {
 
 // NewListerWatcher returns a cache.ListerWatcher for the provided resource.Schema that uses the given ListWatchClient.
 // The List and Watch requests will always use the provided namespace and labelFilters.
-func NewListerWatcher(client ListWatchClient, sch resource.Schema, namespace string, labelFilters ...string) cache.ListerWatcher {
+func NewListerWatcher(client ListWatchClient, sch resource.Schema, namespace string, labelFilters []string, fieldSelectors []string) cache.ListerWatcher {
```

Comment:
I think it might be better for us to define some type like 
```go
type ListWatchOptions struct {
    LabelSelectors []string
    FieldSelectors []string
}
```
And consume that, rather than adding to the argument list. We could also use this in the New methods like `NewKubernetesBasedInformerWithFilters` and `NewMemcachedInformerWithLabelFilters`

Reply:
Agree, I would also move the namespace to the options as well. IIRC this is how it's expressed in k8s stdlib too.

Reply:
Just saw your comment @radiohead,  I'll add that also

Reply:
Now we have `simple.ListWatchOptions` and `operator.ListWatchFilterOptions`  that are duplicates. 

Thoughts?

Reply:
@bcotton I would remove `simple.ListWatchOptions` OR make it a type alias. @IfSentient WDYT?

Reply:
Moved `simple.ListWatchOptions` to `operator.ListWatchOptions` and removed `operator. ListWatchFilterOptions`